Core/Core/View/Validator.swift

```diff
@@ -9,24 +9,23 @@ import Foundation
 
 public class Validator {
     
-    private let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
+    private let emailRegEx = ".+@.+\\..+"
```

Comment:
What is the reason for changing this regex expression?

Reply:
reverted it back to original.